tests/rest/admin/test_user.py

```diff
@@ -2080,6 +2239,36 @@ def _check_fields(self, content):
             self.assertIn("quarantined_by", m)
             self.assertIn("safe_from_quarantine", m)
 
+    def _order_test(
+        self,
+        order_by: Optional[str],
+        dir: Optional[str] = None,
+        expected_media_list: List[str],
+    ):
+        """Request the list of media in a certain order. Assert that order is what
+        we expect
+        Args:
+            order_by: The type of ordering to give the server
+            dir: The direction of ordering to give the server
+            expected_media_list: The list of media_ids in the order we expect to get
+                back from the server
+        """
+
+        url = self.url + "?"
+        if order_by is not None:
+            url += "order_by=%s&" % (order_by,)
+        if dir is not None and dir in ("b", "f"):
+            url += "dir=%s" % (dir,)
+        channel = self.make_request(
+            "GET", url.encode("ascii"), access_token=self.admin_user_tok,
+        )
+        self.assertEqual(200, channel.code, msg=channel.json_body)
+        self.assertEqual(channel.json_body["total"], len(expected_media_list))
+
+        returned_order = [row["media_id"] for row in channel.json_body["media"]]
+        self.assertListEqual(expected_media_list, returned_order)
```

Comment:
`assertListEqual` is used automatically when comparing lists via `assertEqual`.

```suggestion
        self.assertEqual(expected_media_list, returned_order)
```